master/pom.xml

```diff
@@ -2046,6 +2046,12 @@
         <version>${sakai.version}</version>
         <scope>provided</scope>
       </dependency>
+      <dependency>
```

Comment:
Is this related to this PR? A fix for a different PR?

Reply:
Belongs to cardgame but I found this issue when modifying the webapi pom.xml

Reply:
Take it outside, i've fixed it!